shark_turbine/aot/exporter.py

```diff
@@ -252,6 +252,12 @@ def export(
             )
         )
 
+    if not strict_export:
+        warnings.warn(
+            "When strict_export == False, Python interpreter is used and code will execute exactly as it would in eager mode. "
+            "Therefore, graph optimizations and safety are not garaunteed to be at the same level as TorchDynamo."
```

Comment:
Please add "This is an experimental feature in PyTorch that the IREE Turbine project is still evaluating. Please report issues or experiences."

Reply:
Done